_examples/basic/2-realtime-feed/producer/main.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"time"
+
+	"github.com/brianvoe/gofakeit"
+
+	"github.com/ThreeDotsLabs/watermill"
+	"github.com/ThreeDotsLabs/watermill-kafka/pkg/kafka"
+	"github.com/ThreeDotsLabs/watermill/message"
+	"github.com/ThreeDotsLabs/watermill/message/router/middleware"
+)
+
+var (
+	brokers = []string{"kafka:9092"}
+
+	messagesToPublish = 10000
+	workers           = 25
+)
+
+func main() {
+	logger := watermill.NewStdLogger(false, false)
+	logger.Info("Starting the producer", watermill.LogFields{})
+
+	rand.Seed(time.Now().Unix())
+
+	publisher, err := kafka.NewPublisher(brokers, kafka.DefaultMarshaler{}, nil, logger)
+	if err != nil {
+		panic(err)
+	}
+	defer publisher.Close()
+
+	messagePublished := make(chan struct{})
+	allMessagesPublished := make(chan struct{})
+
+	go func() {
```

Comment:
after some thinking: maybe we should remove logic to produce X messages, and instead produce X messages per second, forever?

Reply:
it may simplify it a bit

Reply:
also, proposition from #121 is nice :+1:

Reply:
Yeah that makes sense, as now the producer publishes all of them quickly and then gets restarted by docker-compose. 👍

Reply:
@maclav3 can you handle this one in #121?

Reply:
done, merged

Reply:
VERY NICE, I LIKE